bin/import_ccef.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/env python
+"""
+Importing data from XML provided by Conference Centre for Economics, France
+"""
+
+import csv
+import os
+import progressbar
+import subprocess
+from multiprocessing import Pool
+
+import decimal
+import math
+import unicodedata
+import re
+import sox
+import xml.etree.ElementTree as ET
+from num2words import num2words
```

Comment:
maybe do a try-catch ImportError and document this external dependency? It's not documented elsewhere.

Reply:
@reuben Right, but we don't have a canonical place for that kind of documentation, do we?

Reply:
Sorry, I wasn't clear, I meant try-catch and then print a message telling the user to install num2words.